src/containers/CreateDoc/createDoc.css

```diff
@@ -0,0 +1,53 @@
+.page__content {
```

Comment:
just to double check>  is  `page` your block over here ? and `content` an element inside that block  ?